updater/metadata_fetchers/nvd/nvd.go

```diff
@@ -224,3 +230,22 @@ func getHashFromMetaURL(metaURL string) (string, error) {
 
 	return "", errors.New("invalid .meta file format")
 }
+
+// scoreToPriority converts the CVSS Score (0.0 - 10.0) into the user-friendy
+// types.Priority using the usually reference correlation table, extended
+// for the Negligible level.
+func scoreToPriority(score float64) types.Priority {
```

Comment:
Worth mentioning that this follows the specification?
  https://www.first.org/cvss/specification-document

See table 14